src/App.tsx

```diff
@@ -154,7 +154,7 @@ const App: FC = () => {
         <Footer />
       </StyledApp>
       {!localStorage.getItem('previouslyLoggedIn') && !isLoadingAuthorities && authorityDataUpdated && (
```

Comment:
vil ikke modalen sprette opp et lite øyeblikk før den lukkes igjen nå? kunne lagt til ` && !user.authority` her for å unngå det